realsense_ros2_camera/src/realsense_camera_node.cpp

```diff
@@ -72,9 +72,15 @@ const std::vector<std::vector<stream_index_pair>> IMAGE_STREAMS = {{{DEPTH, INFR
 
 const std::vector<std::vector<stream_index_pair>> HID_STREAMS = {{GYRO, ACCEL}};
 
+rs2::device _dev;
```

Comment:
Is `_dev` now a global variable? Not sure having a global variable is a good practice

Reply:
this is just a temporary solution, we will rebuild the code in the future.:) thanks lalit @lbegani